ONSdigital/babbage
# update tag replacer for the new warning block

### What

Update tag replacer for the new warning block.
https://trello.com/c/3MvV4GNj/3615-implement-warning-block-13
https://github.com/ONSdigital/florence/pull/312
https://github.com/ONSdigital/sixteens/pull/139

### How to review

florence: 
* Edit content
* Add warning box
* Save
Check content against 
https://bulletin-26.netlify.com/sprint/5/warning-text

### Who can review

all
